crates/dbsp/src/trace/ord/file/indexed_zset_batch.rs

```diff
@@ -38,6 +38,37 @@ where
     pub layer: FileOrderedLayer<K, V, R>,
 }
 
+impl<Other, K, V, R> PartialEq<Other> for FileIndexedZSet<K, V, R>
```

Comment:
This looks like a dangerous trait, since this function looks cheap, but could be very dangerous. Who needs it? I could use it in my tests... Except that in general comparing ZSets with FP values does not really work.

Reply:
It's only needed in tests. I could add `#[cfg(test)]` if you'd like.

The existing batch types already support equality tests, it's just that they're limited to comparing against the exact same type.

Reply:
Yes, I think #[cfg(test)] is a good idea. We don't want this to be invoked by mistake.